lib/index.js

```diff
@@ -5,5 +5,6 @@ import { NativeModules } from 'react-native'
 // NativeModules.ShareExtension.close()
 export default {
   data: () => NativeModules.ReactNativeShareExtension.data(),
-  close: () => NativeModules.ReactNativeShareExtension.close()
+  close: () => NativeModules.ReactNativeShareExtension.close(),
+  openURL: (url) => NativeModules.ReactNativeShareExtension.openURL(url),
```

Comment:
Are we able to pass values from the extension to the main app using `openURL`? It doesn't appear that way. What would be the point of redirecting without passing data?

Reply:
You would use the URI sceme of the app with a query string of data that can be read with Linking (deeplink)